library-management-system/src/main/java/com/johnny/training/Book.java

```diff
@@ -0,0 +1,58 @@
+package com.johnny.training;
+
+public class Book {
```

Comment:
If the class won't be extended, make sure to declare it as final. Always favor immutability over mutability in the code (avoids nasty bugs in the future)